Gopkg.toml

```diff
@@ -0,0 +1,79 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+
+required = [
+  "github.com/Azure/go-autorest/autorest",
+  "github.com/docker/distribution/reference",
+  "k8s.io/api/core/v1",
+  "k8s.io/apimachinery/pkg/apis/meta/v1",
+  "k8s.io/apiserver/pkg/endpoints/request"
+]
+
+[[constraint]]
+  name = "github.com/Azure/go-autorest"
+  version = "9.1.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/docker/distribution"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/e2esetup"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/e2etests"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/helmclient"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/micrologger"
+
+[[constraint]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.13.2"
```

Comment:
Deps are outdated. They should be `1.15.4`. 

You can base changes on this. https://github.com/giantswarm/metrics-server-app/blob/7bea72cd0f5e50d591db7856cb6e40b5105a7ee4/Gopkg.toml